docs/docs-content/clusters/edge/site-deployment/cluster-deployment.md

```diff
@@ -92,7 +95,11 @@ Use the following steps to create a new host cluster so that you can add Edge ho
    the **control-plane-pool** and the **worker-pool**. When you have completed configuring the node pools, click on
    **Next**.
 
-10. (Optional) When you assign Edge hosts to node pools, you can optionally specify a static IP address for each Edge
+10. (Optional) If you want to provision a two-node high availability cluster, check the `**Enable Two-Node Capability`
```

Comment:
```suggestion
10. (Optional) If you want to provision a two-node high availability cluster, check the **Enable Two-Node Capability**
```

Reply:
I'm assuming you wanted this to be bold, but feel free to change it back to backticks.